### ISSUE TITLE
Improve character escaping

### ISSUE THREAD

**Issue by Drugantibus** [timbru31/mp3lyrics#155]
mp3lyrics works great for most songs, but seems to miss every song in which the title contains characters like `( ) - .` and probably others. I imagine this has to do with character escaping. Since especially parentheses are pretty common in song titles, this results in about a 35% miss in my experience.

**Comment by timbru31** [comment]
Could you share some examples of titles that did not work? That helps identifying the issue(s).

**Comment by Drugantibus** [comment]
Sure! Artist and title are as reported by mp3lyrics (`fetching lyrics for ...`)

```
Logic - Mama / Show Love (feat. YBN Cordae)
Logic - Still Ballin (feat. Wiz Khalifa)
Joey Bada$$ - Y U DON'T LOVE ME? (MISS AMERIKKKA)
Articolo 31 - D.J jad
Aminé - Can’t Decide
```

It actually seems like parentheses alone are not a problem, since i.e. `Logic - Lucidity (Scene)` fetched correctly; it seems that it's the dot in `feat.` that causes problems.

**Comment by timbru31** [comment]
Hi,

I've tried an example .mp3 file which contains a `.` in it's file name and the corresponding title field and mp3lyrics found the song correctly.

Can you share some more information such as your OS and ruby version? Are you running/using a specific wiki service?

**Comment by Drugantibus** [comment]
Hi, OS is Arch Linux, ruby version I'm not sure since I cannot power on the machine where I have mp3s and mp3lyrics at the moment, but since Arch is rolling I guess it was the latest version available at the time of the issue creation. I usually force fetching from Genius.

**Comment by timbru31** [comment]
Thanks, that helps to narrows the issue down. 👍

**Comment by timbru31** [comment]
I've published v0.1.2 which addresses some of your issues - feel free to give it a second try on your media center!

**Comment by timbru31** [comment]
Closing as this should have been fixed. If not, please comment and I'll re-open this bug.
